frb/galaxies/hosts.py

```diff
@@ -84,3 +88,102 @@ def random_separation(catalog, wcs, npix, trim=1*units.arcmin, ntrial=100):
     idx, d2d, d3d = match_coordinates_sky(grid_coord, cat_coord, nthneighbor=1)
 
     return d2d
+
+
+def get_R(R_frb, R_0=0.2, R_h=0.25):
+    """
+    Calculates Radius of localisation region in arcsecond
+    Based on Bloom et al 2002 and Eftekhari et al 2017
+    
+    Args:
+        R_frb: The 1 sigma localization radius of the FRB
+        R_0: Radial angular separation between the FRB position and a presumed host
+        R_h: Galaxy half light radius
+    
+    Returns:
+        float: radius (in arcseconds)
+    """
+    return np.max([2*R_frb, np.sqrt(R_0**2 + 4*R_h**2)])
+
+
+def read_r_mags(data_table_path):
+    """
+    Reads data used in Driver et al (2016).
+    https://iopscience.iop.org/article/10.3847/0004-637X/827/2/108
+
+    Args:
+        data_table_path: Path to the fits file with data 
```

Comment:
If this file is relatively small in size (e.g. <10Mb), please
add in somewhere sensible under `data/` and then you 
can load it directly from the repository.

Reply:
Ah, I see you do so below.  Ignore this then.  :)